features/git-ship/current_branch/on_feature_branch_without_open_changes.feature

```diff
@@ -38,8 +38,10 @@ Feature: git ship: shipping the current feature branch
   Scenario: feature branch with non-pulled updates in the repo
     Given I have a feature branch named "feature"
     And the following commit exists in my repository
-      | BRANCH  | LOCATION | FILE NAME    | FILE CONTENT    |
-      | feature | remote   | feature_file | feature content |
+      | BRANCH  | LOCATION         | FILE NAME    | FILE CONTENT          |
```

Comment:
These are related. Without these additional commits, its the coworker who is the only with a commit on the branch. This is because we implement remote commits as being a coworker committing them instead of the generic user committing them from another machine